test/integration/crcsuite/collect.go

```diff
@@ -188,7 +188,7 @@ func (collector *VMCommandCollector) Collect(w Writer) error {
 	if err != nil {
 		return err
 	}
-	out, err := ssh.Output(collector.Command)
+	out, _, err := ssh.Run(collector.Command)
 	if err != nil {
```

Comment:
These changes in `collect.go` belong in the previous commit, I've already fixed it locally.

Reply:
Fixed in the latest push.